environs.py

```diff
@@ -228,12 +228,16 @@ def read_env(
                 raise RuntimeError("Could not get current call frame.")
             frame = current_frame.f_back
             caller_dir = os.path.dirname(frame.f_code.co_filename)
+            # Will be a directory
             start = os.path.join(os.path.abspath(caller_dir))
         else:
+            # Could be directory or a file
             start = path
         if recurse:
+            # Reaching this point, 'start' MUST exist. If it is a file, 'os.path.dirname' will be used.
```

Comment:
This looks inaccurate, since `dirname` isn't used. We can probably just remove this comment, since the below line is self-explanatory.